trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -95,7 +95,7 @@ def plot_vehicle(vehicle, save_figure = False, plot_control_points = True, save_
     propulsor_edge_color = 'black' 
     propulsor_alpha      = 1    
     for propulsor in vehicle.propulsors:    
-        plot_propulsor(axes,VD,propulsor,propulsor_face_color,propulsor_edge_color,propulsor_alpha)    
+        plot_propulsor(axes,propulsor)    
```

Comment:
What is the reasoning behind these changes?

Reply:
They just weren't being used in the plot_propulsor function